### ISSUE TITLE
Removes new line at the end of the file

### ISSUE THREAD

**Issue by thewilkybarkid** [ts-graphviz/prettier-plugin-dot#3]
## Describe the bug

it's common to have a new line at the end of files to avoid a 'No newline at end of file' message when diffing; the plugin currently removes it.

## To Reproduce

Run the Prettier fixer on a file that has a trailing new line:

```graphviz
digraph G {
  Hello -> World;
}

```

The trailing new line is removed, becoming:

```graphviz
digraph G {
  Hello -> World;
}
````

## Expected behavior

It's left unchanged.

**Comment by kamiazya** [comment]
@all-contributors please add @thewilkybarkid  for bug

**Comment by kamiazya** [comment]
@thewilkybarkid 
This bug is fixed in [v0.3.0](https://github.com/ts-graphviz/prettier-plugin-dot/releases/tag/v0.3.0).

Thank you!

**Comment by thewilkybarkid** [comment]
@kamiazya Thanks! And thanks for the plugin too! 😃
